bridge/slack/users_channels.go

```diff
@@ -87,7 +87,6 @@ func (b *users) populateUser(userID string) {
 			// in case the previous query failed for some reason.
 		} else {
 			b.usersSyncPoints[userID] = make(chan struct{})
-			b.usersMutex.Unlock()
```

Comment:
This should actually be kept but the `break` on the line below should be a `return` instead.